README.md

```diff
@@ -20,48 +20,44 @@ The [fal](https://github.com/fal-ai/fal) + [dbt-fal](https://github.com/fal-ai/f
 
 With this combo, you won't have to leave your dbt project and still add more capabilities to your stack.
 
+### Pre-requisites
+
+- [Install Docker](https://docs.docker.com/get-docker/). Check if the installation is ok with `docker --version`.
+  - **Note:** if you're on Linux, check the [post-install steps](https://docs.docker.com/engine/install/linux-postinstall/).
+- Make sure `dbt-core` is installed. Verify with `dbt --version`. If not, see [DBT Installation](https://docs.getdbt.com/docs/get-started/installation) for instructions.
```

Comment:
`dbt-core` is not recommended to be installed by itself, the idea would be to point people to install `dbt-postgres` (which installs dbt-core itself, but it's not something we need to mention)

Reply:
Thanks for the callout, I guess the right wording would be _make sure DBT is installed_.

Also, now that I'm thinking, wouldn't `pip install -r requirements.txt`, which contains `dbt-postgres`, result in the dbt cli added to the path if it wasn't before? I didn't try that.

Reply:
> Also, now that I'm thinking, wouldn't pip install -r requirements.txt, which contains dbt-postgres, result in the dbt cli added to the path if it wasn't before? I didn't try that.

yes it would!